src/Http/Controllers/CP/Users/PasswordController.php

```diff
@@ -31,6 +34,10 @@ public function update(Request $request, $user)
 
         UserPasswordChanged::dispatch($user);
 
+        if ($currentPassword = $request->current_password) {
```

Comment:
@duncanmcclean Snappy AF. But tittle typo here, should be `!==` I guess.

Reply:
I'm just assigning the `current_password` to a variable here, not doing any kind of if statement.

Reply:
It's fine the way he has it. The `logoutOtherDevices` method needs the current password, so he's putting it in a variable. The current_password field would only have been submitted if you were editing your own password, so it's wrapped in a condition.

Reply:
All good!